e2e/test/scenarios/embedding/embedding-reproductions.cy.spec.js

```diff
@@ -504,106 +501,182 @@ describe("issues 20845, 25031", () => {
   });
 });
 
-describe.skip("27643", { tags: "@external" }, () => {
+// TODO:
+// - Add tests for embedding previews in both cases
+// - Add tests for disabled, editable and locked parameters in both cases
+// BONUS: Ideally add tests for email subscriptions with the filter applied
+describe("issue 27643", () => {
   const PG_DB_ID = 2;
+  const TEMPLATE_TAG_NAME = "expected_invoice";
+  const getQuestionDetails = fieldId => {
+    return {
+      name: "27643",
+      database: PG_DB_ID,
+      native: {
+        query: "SELECT * FROM INVOICES [[ where {{ expected_invoice }} ]]",
+        "template-tags": {
+          [TEMPLATE_TAG_NAME]: {
+            id: "3cfb3686-0d13-48db-ab5b-100481a3a830",
+            dimension: ["field", fieldId, null],
+            name: TEMPLATE_TAG_NAME,
+            "display-name": "Expected Invoice",
+            type: "dimension",
+            "widget-type": "string/=",
+          },
+        },
+      },
+      enable_embedding: true,
+      embedding_params: {
+        [TEMPLATE_TAG_NAME]: "enabled",
+      },
+    };
+  };
 
   beforeEach(() => {
     restore("postgres-12");
     cy.signInAsAdmin();
     resyncDatabase({
       dbId: PG_DB_ID,
     });
-    const TEMPLATE_TAG_NAME = "boolean_param";
     withDatabase(PG_DB_ID, ({ INVOICES }) => {
-      /** @type {import("e2e/support/helpers").NativeQuestionDetails} */
-      const questionDetails = {
-        name: "Postgres SQL with boolean parameter",
-        database: PG_DB_ID,
-        native: {
-          query: "SELECT * FROM INVOICES [[ where {{ boolean_param }} ]]",
-          "template-tags": {
-            boolean_param: {
-              id: "3cfb3686-0d13-48db-ab5b-100481a3a830",
-              dimension: ["field", INVOICES.EXPECTED_INVOICE, null],
-              name: TEMPLATE_TAG_NAME,
-              "display-name": "Boolean Param",
-              type: "dimension",
-              "widget-type": "string/=",
-            },
-          },
+      cy.wrap(INVOICES.EXPECTED_INVOICE).as(
```

Comment:
we need to still use Postgres to test as [stated in the issue](https://github.com/metabase/metabase/issues/27643) that the default H2 doesn't have this problem.